proposals/mips/mip-o14/MTokens.json

```diff
@@ -0,0 +1,23 @@
+{
+    "10": [
+        {
+            "addressesString": "MOONWELL_USDT0",
+            "borrowCap": 900000e6,
+            "collateralFactor": 0.83e18,
+            "initialMintAmount": 1e6,
```

Comment:
@Chrizzled we will need to put at least $1 of USDT0 in the temporal governor prior to passage.